client/components/loadable/style.scss

```
@@ -0,0 +1,26 @@
+.is-loadable-placeholder {
+	animation: loading-fade 1.6s ease-in-out infinite;
+	background-color: '#e2e4e7';
```

```suggestion
	background-color: #e2e4e7;
```

But could we maybe import `@wordpress/base-styles` where the `$light-gray-500` and other colors come from instead?

---

This is done.